### PR TITLE
Lg/community contrib topics

### PR DESCRIPTION
**Pull request by lisa-parity**

### CODE REVIEW

**Review by tarakj — APPROVED** [review]
lgtm

File: content/md/en/docs/community/article-bounties.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  - style guide
+  - bounties
+---
+
+<Message
+  type="yellow"
+  title="Information"
+  text={`This section is still work in progress. There is no official Substrate Developer
+Hub bounty program although we plan to introduce one soon.`}
+/>
```
Comment (sacha-l) [inline content/md/en/docs/community/article-bounties.md:18]:
```suggestion
This section is still work in progress. There is no official Substrate Developer Hub bounty program although we plan to introduce one soon.
```

File: content/md/en/docs/community/contributor-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+For simple changes, like fixing a typo or making minor changes to a sentence:
+
+1. Click **Edit this page**.
+
+2. Edit the page in GitHub.
+
+3. Replace the default commit message with a short description of your change, then click **Commit Changes**.
+
+4. Select **Create a new branch for this commit and start a pull request**.
```
Comment (sacha-l) [inline content/md/en/docs/community/contributor-guidelines.md:65]:
```suggestion
1. Click **Edit this page**.

1. Edit the page in GitHub.

1. Replace the default commit message with a short description of your change, then click **Commit Changes**.

1. Select **Create a new branch for this commit and start a pull request**.
```

File: content/md/en/docs/community/contributor-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+   ```
+
+1. Click **Create pull request** to start a new pull request and provide any additional information about the changes you made.
+    
+    A maintainer will review your pull request and approve or request changes.
+    If no changes are required, the maintainer will merge your pull request.
+    If a maintainer requested changes or clarification, update your pull request and request another reviewer.
+
+1. When you see your changes have been merged, celebrate your success!
+   🥂
```
Comment (sacha-l) [inline content/md/en/docs/community/contributor-guidelines.md:109]:
```suggestion
8. Click **Create pull request** to start a new pull request and provide any additional information about the changes you made.
    
    A maintainer will review your pull request and approve or request changes.
    If no changes are required, the maintainer will merge your pull request.
    If a maintainer requested changes or clarification, update your pull request and request another reviewer.

9. When you see your changes have been merged, celebrate your success!
   🥂
```

Comment (sacha-l) [reply content/md/en/docs/community/contributor-guidelines.md:109]: The numbering is acting up so I just changed the last two here.

File: content/md/en/docs/community/contributor-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+Only the information that is pertinent to achieving that goal or completing the task is included. 
+With how-to guides, readers have enough information to know what they want to do—for example, open a bank account—but not necessarily enough information to know how to do it.
+For example, the how-to guide for opening a bank account wouldn't explain what a bank account is or why you might want to open one, but would focus on specific steps such as:
+
+1) Select an institution.
+2) Fill out an application.
+3) Deposit a minimum amount of currency.
+ 
+How-to guides often include links to additional information, but should not include explanations that take the focus away from what the reader wants to accomplish.
+For more information about writing _How-to_ guides, see the [Template - How-to guide](/community/template-how-to-guide) and the [Markdown template](https://github.com/substrate-developer-hub/substrate-docs/blob/main/static/assets/contribute-templates/how-to-template.md).
```
Comment (sacha-l) [inline content/md/en/docs/community/contributor-guidelines.md:129]:
```suggestion
For more information about writing _How-to_ guides, see the [Template - How-to guide](/community/template-how-to-guide) and the [Markdown template](static/assets/contrib-templates/how-to-template.md).
```

**Review by sacha-l — COMMENTED** [review]
Glad to see a home for this material!

**Pull request merged**